Public/Src/Engine/Processes/SandboxedProcess.cs

```diff
@@ -492,6 +492,8 @@ private async Task OnProcessExitingAsync()
 
         private async Task OnProcessExited()
         {
+            m_processIdListener?.Invoke(-ProcessId);

```

Comment:
>m_processIdListener [](start = 12, length = 19)

Does it have to go as low as `SandboxedProcess`? Can it be tracked in `SandboxedProcessPipExecutor`?

Reply:
Good point.  It's more accurate if it's here, but I don't think it needs to be that accurate, so I'll move it.

Reply:
done.